src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,18 +11,26 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
 
+	private String _feedbackToUser;
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.